sample.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't give sample files for other features so let's keep it consistent by removing `sample.json` or adding it to `.gitignore`